### PR TITLE
add non-code contributions to contrib guidelines readme

### PR DESCRIPTION
**Pull request by sarahciston**
Requested by Q and Ken, inserted edited text for non-code ways to contribute at bottom of README.md

@all-contributors please add @sarahciston for [doc]

### CODE REVIEW

File: contributor_docs/README.md [hunk 1]
```diff
@@ -36,10 +36,17 @@ Head over to [this link](./contributor_guidelines.md) where you will be guided o
...
 
 ## Non-source code contribution
-There are many more ways to contribute to p5.js through non-source code contribution than can be exhaustively listed here, some of the ways may also involve working with some of the p5.js repositories (such as adding examples, writing tutorials for the website, etc.). Depending on what the planned contribution is, we may be able to support you in different ways so do reach out to us via any channel available to you (email, social media, [Discourse forum](https://discourse.processing.org/c/p5js/10), Discord, etc).
+There are many more ways to contribute to p5.js through non-source code contribution than can be exhaustively listed here. Some of the ways may also involve working with some of the p5.js repositories (such as adding examples, writing tutorials for the website, etc.). Depending on what the planned contribution is, we may be able to support you in different ways so do reach out to us via any channel available to you (email, social media, [Discourse forum](https://discourse.processing.org/c/p5js/10), Discord, etc). Here are just some ways you can contribute:
```
Comment (Qianqianye) [inline contributor_docs/README.md:42]: Hi @sarahciston, we mentioned earlier that documentation is part of Source-code contribution, so we will need to move the documentation related text up. What do you think?

CC @limzykenneth

**Comment by Qianqianye** [comment]
Hi @limzykenneth, because the README.md file is quite long with the details about source-code contribution. I'm not sure if it still makes sense to have it as the intro text on the contribute page for the new website. The backup plan is to have this README.md file as an individual contributor doc item. What do you think?

File: contributor_docs/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+Thanks for your interest in contributing to p5.js! p5.js is a collaborative project with contributions from many volunteers. Our community is always looking for contributors and appreciates involvement in all forms. We acknowledge that not everyone has the capacity, time, or financial means to participate actively or in the same ways. We want to expand the meaning of the word “contributor.” Whether you're an experienced developer or just starting out, we value your involvement. Your unique perspectives, skills, and experiences enrich our community, and we encourage you to get involved in a way that works for you. It includes documentation, teaching, writing code, making art, writing, design, activism, organizing, curating, or anything else you might imagine. Our [contribute page](https://p5js.org/contributor-docs/#/) gives an overview of different ways to get involved and contribute.
```
Comment (Qianqianye) [inline contributor_docs/README.md:3]: @sarahciston I made some edits on some other paragraphs to combine the new writing you had and some existing text. Please feel free to make edits.

File: contributor_docs/README.md [hunk 3]
```diff
@@ -17,8 +17,8 @@ Next, we are currently prioritizing work that expands access (inclusion and acce
 
 # Get Started
 Now you are ready to start contributing to p5.js! There are many ways to get started with contributing to p5.js and many reasons to do so. For the purpose of this documentation, we will split contributions roughly into two categories.
-- Contributions that directly deals with the source code (including documentation)
-- Contributions that directly deals with the source code very little or not at all
+- Source code contribution (including documentation)
+- Non-source code contribution
```
Comment (Qianqianye) [inline contributor_docs/README.md:22]: I change the two category title to the chapter titles. If it's making things easier, we can remove 'including documentation' in 'Source code contribution,' and include it in 'Non-source code contribution' chapter.

**Comment by limzykenneth** [comment]
We'll still need copy for the contribute page in any case, if this one is too long we can ignore it for the website and just let it live in this repo only and have something shorter on the website.

File: contributor_docs/README.md (hunk 1 above)
Comment (sarahciston) [reply contributor_docs/README.md:42]: I agree, and in that case we'd want to provide more introduction to documentation contributions there as well, right? On the website contribution page it says, "Document. Everyone loves helpful instructions. Check out the p5.js [GitHub](https://github.com/processing/p5.js) on how to get started with creating examples, improving reference pages, and writing tutorials." So where can we point to specifically for showing how to get started with writing reference pages and docs?

File: contributor_docs/README.md (hunk 3 above)
Comment (sarahciston) [reply contributor_docs/README.md:22]: I like keeping "including documentation". And would also like to add to 'non code contributions' (see below) so it's more clear where to find out about that.

**Review by sarahciston — COMMENTED** [review]
Made a couple short comments for possible ideas. Otherwise looks good to me.

File: contributor_docs/README.md (hunk 1 above)
Comment (Qianqianye) [reply contributor_docs/README.md:42]: We can point them to Contribute page on the new website

Comment (sarahciston) [reply contributor_docs/README.md:42]: On the Contribute page on the new website it points them to the GitHub contributor docs. Is there a place in the docs that addresses creating documentation specifically? Maybe we could point them to that? If not, perhaps that's something we could build in the future if it's desirable, but may need to remove the description of the pointer from the website if we don't have info on how to get started writing documentation yet. What do you both think? @Qianqianye @limzykenneth

Comment (Qianqianye) [reply contributor_docs/README.md:42]: We have a doc about how to write referenece, and upcoming doc about how to write examples. We will have new links when the new website is updated. Maybe we can just mention the doc generically without links for now?

**Pull request merged**